poppy/matrixDFT.py

```diff
@@ -302,22 +263,10 @@ def DFT_fftstyle_rect(pupil, nlamD, npix, inverse=False):
     t2 = np.dot(t1, expXU)
 
     if inverse:
-        #print expYV.shape, pupil.shape, expXU.shape
         t2 = t2[::-1, ::-1]
-    #else:
-        #expYV = expYV.T.copy()                          # matrix npixY * npupY
-        #t1 = np.dot(expYV, pupil)                       # matrix npixY * npupX?     dot prod combined last axis of expYV and first axis of pupil.
-        #t2 = np.dot(t1, expXU)                          # matrix npixY * npixX
-        #print expYV.shape, pupil.shape, expXU.shape
-
-    #print ""
-
-    #return  nlamD/(npup*npix) *   t2 * dx * dy
     # normalization here is almost certainly wrong:
```

Comment:
Conservation of energy across the FFT.   The total intensity in the output plane should be equal to that in the input plane. (Perfectly true for infinitely large arrays sampled continuously; approximately true for any finite sampling)

I looked through the test code to refresh myself on this. It looks like test_matrixDFT's test_MFT_flux_conservation and test_MFS_fluxconsv_all_types verifies this. The tests pass for all of the flavors of matrixDFT, so I think the comment is out of date and the normalization is correct.